Makefile

```diff
@@ -1,9 +1,11 @@
 PYTHON := python3
 COMPILER_LAUNCHER ?= $(shell which ccache)
 COVERAGE := --cov=pennylane_lightning --cov-report term-missing --cov-report=html:coverage_html_report
-TESTRUNNER := -m pytest tests --tb=short
+TESTRUNNER := -m pytest tests --tb=short -x -vvv -s
```

Comment:
Do we want to add `-x` as a default option? Seems like it could slow us down if we want to get a report with multiple failures

Reply:
I always add it locally as it improves my debugging process. If you don't share this feeling, I can remove it.
Thanks for your feedback.